src/api-routes/persons/{osuId}/addresses.js

```diff
@@ -0,0 +1,31 @@
+import { errorHandler, errorBuilder } from 'errors/errors';
+import { getAddressesById } from 'db/oracledb/addresses-dao';
+
+/**
+ * Get addresses by OSU ID
+ *
+ * @type {RequestHandler}
+ */
+const get = async (req, res) => {
+  try {
+    const { osuId } = req.params;
+    const { query } = req;
+    const result = await getAddressesById(osuId, query);
+    if (!result) {
```

Comment:
Same thing as images, result not found can possible be "the person exist but he doesn't have address or the filtered address doesn't exist". In this case, you need to return a 200 empty array instead of a 404.

You can reproduce the issue by searching `/persons/:osuId/addresses?filter[city]=foo`